app/src/main/res/layout/news_holder.xml

```diff
@@ -78,14 +80,12 @@
                 app:layout_constraintStart_toStartOf="@+id/guideline2"
                 app:layout_constraintTop_toBottomOf="@+id/new_description" />
 
-
-        <ImageView
+        <CheckBox
                 android:id="@+id/favorite_icon"
                 android:layout_width="20dp"
                 android:layout_height="20dp"
-                android:layout_marginEnd="15dp"
-                android:layout_marginBottom="10dp"
-                android:background="@drawable/favorites_icon_unfilled"
+                android:layout_marginEnd="20dp"
```

Comment:
it's a bad practice to put constants directly in the layouts. It's better to extract them into dimens.xml

Reply:
gonna fix it as part of a refactor